meteor/client/ui/RundownView/RundownTiming/BreakWallTime.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames'
+import React from 'react'
+import { SegmentUi } from '../../SegmentTimeline/SegmentTimelineContainer'
+import { RundownUtils } from '../../../lib/rundown'
+import { useTranslation } from 'react-i18next'
+
+interface IBreakWallTimeProps {
+	segment: SegmentUi
+	className?: string
+	labelClassName?: string
+}
+
+/**
+ * A presentational component that will render a counter that will show the wall time that a break starts at.
+ * (Or ends at, if there's no start time but an end time is provided.)
+ * @function BreakWallTime
+ * @extends React.Component<IBreakWallTimeProps>
+ */
+export const BreakWallTime = function BreakWallTime(props: IBreakWallTimeProps): JSX.Element | null {
+	const { t } = useTranslation()
+	const value = props.segment.segmentTiming?.expectedStart ?? props.segment.segmentTiming?.expectedEnd
+	const passedExpectedStart =
+		props.segment.segmentTiming?.expectedStart && Date.now() > props.segment.segmentTiming?.expectedStart
+
+	if (value === null || value === undefined) {
+		return null
+	}
+
+	const date = new Date(value)
+
+	return (
+		<>
+			<span className={props.labelClassName}>{t(passedExpectedStart ? 'Planned End' : 'Planned Start')}</span>
+			<span className={classNames(props.className)} role="timer">
+				{RundownUtils.padZeros(date.getHours(), 2)}:{RundownUtils.padZeros(date.getMinutes(), 2)}:
+				{RundownUtils.padZeros(date.getSeconds(), 2)}
```

Comment:
`RundownHeader` currently uses `<Moment interval={0} format="HH:mm:ss" date={expectedEnd} />` to do this sort of thing, I propose we do the same here. NRK has a task to remove all of `Moment` and use something else™️ soon.